### ISSUE TITLE
Brim does not use suricata.rules

### ISSUE THREAD

**Issue by polunika** [brimdata/brimcap#233]
Hi all,

currently I am doing some challenges from cyberdefenders.org. Therefore, I am using Brim to analyze the provided pcaps.
Unfortunately, the listing of Suricata-alerts doesn't work well in my opinion.
When I use the base configuration, Brim shows 8 results with the following filter: **event_type=="alert" | alerts := union(alert.category) by src_ip, dest_ip**
When I  delete the file **suricata.rules** from the directory **/opt/Brim/resources/app.asar.unpacked/zdeps/suricata/var/lib/suricata/rules/**, Brim shows me the same 8 results with the above filter after restarting and importing the pcap again.
Even if I replace the file **suricata.rules** with a provided file from cyberdefenders, Brim shows the same 8 results like before.
From a blog-post (https://ellisstannard.medium.com/malware-traffic-analysis-3-c4c47757e184) I know, that it is possible to list more results with the corresponding filter, but I think I am missing something....

Is it correct, that Brim uses the suricata.rules from the above path automatically or is there another file or directory, that is used?

Thanks in advance

**Comment by polunika** [comment]
Hey.
Thanks for your fast and detailled answer.
Your quick & dirty solution worked fine! I followed your steps meticulously, after I made some faults.
It is important, to place the custom `suricata.rules file` into `$HOME/.config/Brim/data/suricata/rules` AFTER Brim is started and configured automatically. At first I tried it vice versa, but it didn't work: The MD5 Sums were the same in both of the above directories.
People, who are using this solution, should keep in mind, that the `suricata.rules file` in `$HOME/.config/Brim/data/suricata/rules` is changed after each start of Brim.

Thanks for the links. I will go through them and try to do the following (if it is possible): Create a folder with custom `suricata.rules files`. Brim should then use this folder and the automatically updated ones (via ./suricataupdater)
